web_src/js/utils.js

```diff
@@ -161,3 +161,8 @@ export function useLightTextOnBackground(backgroundColor) {
   const brightness = (0.299 * r + 0.587 * g + 0.114 * b) / 255;
   return brightness < 0.35;
 }
+
+// return whether an array has an element
+export function containsEle(arr, element) {
+  return new Set(arr).has(element);
```

Comment:
No .... you shouldn't construct the Set again and again. Otherwise it's slower than `includes`

Reply:
added a commit [868e1ab](https://github.com/go-gitea/gitea/pull/23624/commits/868e1ab2b0bfdbfd5f411c73139c3e5e098a5068) to modify